dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -336,8 +334,7 @@ public void testIsUnusedClassSymbolReference_multiReleaseJar()
       throws IOException, RepositoryException {
     // org.graalvm.libgraal.LibGraal class has different implementations between Java 8 and 11 via
     // Multi-release JAR of this artifact.
-    Artifact grpcArtifact = new DefaultArtifact("org.graalvm.compiler:compiler:19.0.0");
```

Comment:
There were many typos to use `grpcArtifact` variable name for artifacts not related to grpc.